src/Controller/CallbackController.php

```diff
@@ -55,6 +55,19 @@ public function index()
 
         // Store token details
         $this->loadComponent('XeroOauth2.Storage');
+
+        if ($this->Storage->getAccessToken()) {
```

Comment:
`getAccessToken` method will return `null` in case access token is not found, so I would suggest instead of checking for bool we should check if it's `null`.

```suggestion
        if ($this->Storage->getAccessToken() === null) {
```